logging/handlers.rst

```diff
@@ -0,0 +1,106 @@
+Handlers
+========
+
+ElasticsearchLogstashHandler
+----------------------------
+
+.. versionadded:: 4.4
+
+    The ``ElasticsearchLogstashHandler`` was introduced in Symfony 4.4.
+
+This handler dials directly with the HTTP interface of Elasticsearch. This means
```

Comment:
This phrase looks strange to me. Maybe `dials with` should be `deals with`?

Reply:
I don't know. I'm not fluent :) I meant PHP dialog with ES via the HTTP interface. (You can communicate with ES with the "java interface").

Anyway, I updated the sentence with your proposal